src/metabase/lib/js.cljs

```diff
@@ -107,50 +111,36 @@
 (defn ^:export orderable-columns
   "Return a sequence of Column metadatas about the columns you can add order bys for in a given stage of `a-query.` To
   add an order by, pass the result to [[order-by]]."
-  ([a-query]
-   (orderable-columns a-query -1))
-  ([a-query stage-number]
-   (to-array (lib.core/orderable-columns a-query stage-number))))
+  [a-query stage-number]
+  (to-array (lib.core/orderable-columns a-query stage-number)))
```

Comment:
Not really part of this issue... but as mentioned above, a while back we decided to make `stage-number` required and explicit in the TS version of MLv2, so there is no reason to export all of the arities where it's given a default value. I tried to go thru everything and remove all of these unused arities.